chpl/chpl-resources/src/main/resources/directReview.properties

```diff
@@ -0,0 +1,4 @@
+#mapping to Jira custom fields that may change on any given instance of Jira
+directReview.startDate=customfield_10919
+directReview.endDate=customfield_10920
+directReview.circumstances=customfield_10932
```

Comment:
We're using more fields than these, right? Should there be more in this file?

Reply:
Yep, glad you caught this. It was part of my attempt to read those "customfield" names from a properties file since they may change often and be different in different environments. I couldn't get it to work and forgot to delete this stuff.